tests/std/tests/LWG3234_math_special_overloads/test.cpp

```diff
@@ -0,0 +1,573 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <cassert>
+#include <cmath>
+#include <limits>
+#include <type_traits>
+
+using namespace std;
+
+struct Ambiguous : true_type { // ADL

```

Comment:
I have no idea what this comment is trying to tell me.

Reply:
The comment was present because this test file is an imitation of [that of libc++](https://github.com/llvm/llvm-project/blob/main/libcxx/test/std/numerics/c.math/cmath.pass.cpp). Removing it as it might be unhelpful.